libskk/state.vala

```diff
@@ -423,15 +423,34 @@ namespace Skk {
         {
             var command = state.lookup_key (key);
             // check abort and commit event
-            if (command == "abort") {
-                bool retval;
+            if (command == "abort" ||
+                command == "abort-to-latin" ||
+                command == "abort-to-latin-passthrough") {
```

Comment:
There is ```commit-unhandled``` where ```-unhandled``` means the same thing as ```-passthrough``` here.  I rather like the latter, though.